fuse_variables/package.xml

```diff
@@ -12,16 +12,26 @@
   <author email="tmoore@locusrobotics.com">Tom Moore</author>
   <license>BSD</license>
 
-  <buildtool_depend>catkin</buildtool_depend>
-  <depend>libceres-dev</depend>
-  <depend>fuse_core</depend>
-  <depend>pluginlib</depend>
-  <depend>roscpp</depend>
-  <test_depend>roslint</test_depend>
-  <test_depend>rostest</test_depend>
+  <buildtool_depend>ament_cmake_ros</buildtool_depend>
+
+  <build_depend>libceres-dev</build_depend>
+  <build_depend>fuse_core</build_depend>
+  <build_depend>pluginlib</build_depend>
+  <build_depend>rclcpp</build_depend>
```

Comment:
is `rclcpp` required? I don't see it `find_package()`'d in the main CMakeLists.txt - it looks like it's just a `<test_depend>`.

Reply:
I made it a test_depend. Although, `rclcpp` gets included via `fuse_core`; if that's the case should I remove it entirely?
Likewise, [outside of tests, rclcpp::Time gets included and used](https://github.com/methylDragon/fuse/blob/rolling-variables/fuse_variables/include/fuse_variables/acceleration_angular_2d_stamped.h#L85) via `#include <fuse_core/time.hpp>`

https://github.com/locusrobotics/fuse/compare/6b3f4d22c89fe058bceb106625f5bd8ef49c3e40..7a247f8bd9753e0c185d8011cb6f48c537614452

Reply:
Ahh I only looked at the headers. I didn't check for types. Having a `<build_depend>` or `<depend>` on `rclcpp` seems fine to me then!